excercsises/3_synchronization/pages/contact.js

```diff
@@ -8,30 +8,18 @@ var Contact = function () {
   this.getTitle = function () {
     return browser.driver.getTitle();
   };
-  this.fillForm = (name, email, content) => {
-    let nameValue = element(by.id("name")).clear();
-    let emailValue = element(by.id("email"));
-    let contentValue = element(by.id("content"));
-    nameValue.clear();
-    nameValue.sendKeys(name);
-    emailValue.clear();
-    emailValue.sendKeys(email);
-    contentValue.clear();
-    contentValue.sendKeys(content);
-  };
-  this.submitButton = () => {
-    let button = element(by.css(".test"));
-    button.click();
-  };
-  this.checkSucces = (success) => {
-    let expectSuccess = element(by.css(".alert-success"));
+  this.fillFormSubmitCheckSuccess = (name, email, content,success) => {
```

Comment:
We need to take also about it tomorrow - in my opinion, passing here the success information is redundant.